coprocess/python/dispatcher.py

```diff
@@ -143,8 +154,8 @@ def dispatch_event(self, event_json):
     def reload(self):
         tyk.log( "Reloading event handlers and middlewares.", "info" )
 
-        self.purge_event_handlers()
-        self.load_event_handlers()
+        # self.purge_event_handlers()
```

Comment:
Should these be commented out?

Reply:
Event handlers are a separate thing, this doesn't cover them yet.